avocado/utils/ssh.py

```diff
@@ -214,3 +214,37 @@ def quit(self):
         :rtype: bool
         """
         return self._master_command('exit')
+
+    def scp_connection(self):
+        try:
+            SCP_CLIENT_BINARY = path_utils.find_command('scp')
+        except path_utils.CmdNotFoundError:
+            SCP_CLIENT_BINARY = None
+        cmd = self._dash_o_opts_to_str(self.DEFAULT_OPTIONS)
+        cmd = "%s %s" % (SCP_CLIENT_BINARY, cmd)
+        return cmd
+
+    def scp_cmd(self, source, destination, reverse=False, recursive=False):
```

Comment:
IMO, we could have a better name for this method. Something like: `copy` or `copy_files` for instance.

Also, I think that the `reverse` option is confusing because of the name of the args. If the user would like a "reverse" behavior, let the user change the order of args during the call. (i.e: `copy_files("/tmp/foo/bar/*", "192.168.0.1:/tmp/foo/bar/", True)`)

Reply:
@beraldoleal ok, i will change like this.